### DISCUSSION TITLE
ElasticSearch 5.x and skywalking 8.9.1

### DISCUSSION THREAD

**Discussion by G-Aaron** [apache/skywalking#8726] [Q&A] [answered]
is it supported?

**Comment by wu-sheng** [comment, marked as answer]
I guess you are asking ElasticSearch?  No, we are not supporting that.

**Reply by G-Aaron** [reply]
yes

**Reply by wu-sheng** [reply]
As a storage, no, we are not supporting that.

**Reply by G-Aaron** [reply]
where is the list of supported versions?

**Reply by G-Aaron** [reply]
And other middlewares.

**Reply by wu-sheng** [reply]
https://skywalking.apache.org/docs/main/latest/en/setup/backend/backend-storage/#backend-storage

**Reply by G-Aaron** [reply]
thanks.
